### ISSUE TITLE
Finish Antora Docs

### ISSUE THREAD

**Issue by settermjd** [owncloud/docs#92]
## TODO 

- [x] Review labels
- [ ] Create a writing style guide https://github.com/owncloud/docs/issues/95
- [x] Augment the build process https://github.com/owncloud/docs/issues/96
  - [ ] Dead/broken link checker (@mmattel)
  - [x] rST formatting checker (if there is any leftover rST formatting)
- [x] PDF generator (@settermjd, @pvince81)
- [x] ~~Algolia subscription (see @pmaier1)~~
- [x] Implement Site Search https://github.com/owncloud/docs/issues/98
- [x] Update project README & documentation
  - [x] Document difference between when you should use the live preview plugin and when you should use Antora build tools
- [x] New Issue template (including "what needs to be documented (or a description), and "why is this being documented" (or what is the motivation), etc. A good example is https://github.com/owncloud/docs/issues/39 (thanks @voroyam).
- [x] Create redirects for old links

**Comment by Leantricity** [comment]
I'd love to collaborate here. How to start?

**Comment by settermjd** [comment]
Hey @Leantricity, thanks for wanting to contribute! Do you want to help out with this issue specifically, or contribute to the docs as a whole? 

Eitherway, start off by reading [the "Getting Started" guide](https://github.com/owncloud/docs/blob/master/docs/getting-started.md), so that you're abreast of what you need to know to get involved. 

Then, if you have any questions, if anything isn't clear enough, or if the docs could be written better, just let me know. I'm in the #documentation channel in https://talk.owncloud.com/home daily. 

Happy to answer any and all questions!

**Comment by Leantricity** [comment]
Hi Mathew!

I want to collaborate as a whole, mostly if this is a goal, creating the Spanish version of the documentation (I think it does not exist yet).

I will read this Getting Started guide and follow up. I’ve already installed all the Antora required stuff and downloaded the first docs-ui folder as I saw I had to do but seems a demo setup not with the current ownCloud docs.

I will try to learn how to navigate here (I’m new to GitHub also…)

Thanks!

JP
